annotation/vep_annotation.py

```diff
@@ -153,13 +119,56 @@ def get_vep_command(vcf_filename, output_filename, genome_build: GenomeBuild, an
     if settings.ANNOTATION_VEP_ARGS:
         cmd.extend(settings.ANNOTATION_VEP_ARGS)
 
+    if pipeline_type == VariantAnnotationPipelineType.STANDARD:
+        # TODO: At the moment we just skip everything, perhaps we should:
+        # a) Add ColumnVEPField.pipeline_type (or maybe have a list of all types) so we can configure it to run on both
+        # b) Run another pipeline for CNVs
+
+        cmd.extend([
+            # Plugins that don't require data
+            "--plugin", "Grantham",
+            "--plugin", "SpliceRegion",
+        ])
+
+        # Plugins that require data - ok for these to fail when retrieving vep config
+        PLUGINS = {VEPPlugin.MASTERMIND: lambda: f"Mastermind,{vc['mastermind']},1",  # 1 to not filter
+                   VEPPlugin.MAXENTSCAN: lambda: f"MaxEntScan,{vc['maxentscan']}",
+                   VEPPlugin.DBNSFP: lambda: _get_dbnsfp_plugin_command(genome_build, vc),
+                   VEPPlugin.DBSCSNV: lambda: f"dbscSNV,{vc['dbscsnv']}",
+                   VEPPlugin.SPLICEAI: lambda: f"SpliceAI,snv={vc['spliceai_snv']},indel={vc['spliceai_indel']}"}
+
+        if vc.columns_version >= 2:
+            cmd.extend(["--plugin", "NMD"])
+
+        for vep_plugin, plugin_arg_func in PLUGINS.items():
+            try:
+                cmd.extend(["--plugin", plugin_arg_func()])
+            except Exception as e:
+                logging.warning(e)
+                logging.warning("No annotation set for plugin: %s", vep_plugin)
+
+        # Custom
+        for vep_custom, prefix in dict(VEPCustom.choices).items():
+            try:
+                if fields := ColumnVEPField.get_source_fields(genome_build, vep_custom=vep_custom):
+                    prefix_lc = prefix.lower()
+                    if cfg := vc[prefix_lc]:  # annotation settings are lower case
+                        cmd.extend(_get_custom_params_list(fields, prefix, cfg))
+                    else:
+                        logging.info("Skipping due to settings.ANNOTATION[%s][vep_config][%s] = None",
+                                     genome_build.name, prefix_lc)
+            except Exception as e:
+                logging.warning(e)
+                # Not all annotations available for all builds - ok to just warn
+                logging.warning("Skipped custom annotation: %s", prefix)
+
     return cmd
 
 
-def run_vep(vcf_filename, output_filename, genome_build: GenomeBuild, annotation_consortium):
+def run_vep(vcf_filename, output_filename, genome_build: GenomeBuild, annotation_consortium, pipeline_type):
```

Comment:
Can we start making type hints more standard, e.g.
pipeline_type: VariantAnnotationPipelineType